sale_order_project/models/sale.py

```diff
@@ -4,7 +4,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from odoo import api, models, _
-from odoo.exceptions import Warning
+from odoo import exceptions
```

Comment:
You can put this in the first row

Reply:
Well, I meant to put combine with the first row:
```python
from odoo import api, exceptions, models, _
```
But it's not so important.

Reply:
will do on the next one.